### PR TITLE
Add base taxonomy class

### PR DESCRIPTION
**Pull request by snirs90**
#34

### CODE REVIEW

File: plugins/restful/RestfulEntityBaseTaxonomy.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?php
+
+
+/**
+ * @file
+ * Contains RestfulEntityBaseTaxonomy.
```
Comment (amitaibu) [inline plugins/restful/RestfulEntityBaseTaxonomy.php]: Name (and comment) should be `RestfulEntityBaseTaxonomyTerm`

**Comment by amitaibu** [comment]
Instead of this PR, we should have hook_entity_info_alter() and add the `label` property under `entity keys`.

**Comment by amitaibu** [comment]
We actually already have the `label` property, so we can even remove the node base class, and move the logic into `RestfulBase` (`entity_get_info()`...)

**Comment by snirs90** [comment]
The logic of `getPublicFields()` override in `RestfulEntityBaseNode` has been moved to `RestfulEntityBase` and implemented with `entity_get_info()`...

**Comment by amitaibu** [comment]
I've omitted the term class as it's empty, and committed the rest. Thanks.

**Pull request closed without merging**